scss/_breadcrumb.scss

```diff
@@ -46,3 +48,15 @@
     color: var(--#{$prefix}breadcrumb-item-active-color);
   }
 }
+
+// Boosted mod
+.breadcrumb-dark {
+  // scss-docs-start breadcrumb-dark-css-vars
+  --#{$prefix}breadcrumb-color: #{$breadcrumb-dark-color}; // Boosted mod
+  // scss-docs-end breadcrumb-dark-css-vars
+
+  .breadcrumb-item::before {
+    filter: $invert-filter;
```

Comment:
Isn't there a way to do something like `--#{$prefix}breadcrumb-divider-color: #{$breadcrumb-dark-divider-color}` rather than using an invert filter?

In the same spirit, if we change `--bs-breadcrumb-divider-color` value, isn't there a way to make it applied to http://boosted.orange.com/docs/5.2/components/breadcrumb/#example as well? It is difficult to understand that `--bs-breadcrumb-divider-color` is only applied for http://boosted.orange.com/docs/5.2/components/breadcrumb/#dividers.

Reply:
`--#{$prefix}breadcrumb-divider-color` is on Bs side and is only used for non-svg dividers. As said together, we should maybe try to tweak the variable name on BS side to be more consistent with what the variable does.

However, it doesn't work at the moment anyway, so I'll handle that with the next commit.